app/ide-desktop/lib/dashboard/src/authentication/src/error.ts

```diff
@@ -29,6 +33,39 @@ export function tryGetMessage(error: unknown): string | null {
         : null
 }
 
+/** Like {@link tryGetMessage} but returns the string representation of the value if it is not an
```

Comment:
i think we're supposed to avoid singular verbs (verbs ending with `s`), in function docs at least. so `returns` -> `return`, which means this sentence might need to be restructured. maybe "return .... if not an error, like trygetMessage"

Reply:
Done